UM/Qt/Bindings/FileProviderModel.py

```diff
@@ -46,7 +46,8 @@ def update(self) -> None:
             "name"         : "LocalFileProvider",
             "displayText"  : "From Disk",
             "fileProvider" : None,  # it's not loaded via a plugin, so its FileProvider is empty
-            "shortcut"     : "Ctrl+O"
+            "shortcut"     : "Ctrl+O",
+            "priority"     : 0,
```

Comment:
The "From Disk" option should always appear first, so we should increase the priorty of the LocalFileProvider.
```suggestion
            "priority"     : 99,
```